src/ryr-email-templatizer.html

```diff
@@ -33,20 +33,19 @@
       _renderTemplate(templateVars) {
         let template =
           `
-Dear city of Austin,
+          Dear city of Austin,
 
-I would like to have a bike rack here: {{rackLocation | shout |blank>_not provided_}}.
+          I would like to request a bike rack at {{Address}}, Geolocation  {{rackLocation}}. The business supporting this demand is {{businessName}}, located at {{businessAddress}}.
```

Comment:
There is no {{Address}} field (see list [here](https://github.com/request-yo-racks/web/blob/master/src/ryr-email-templatizer.html#L79-L87)).
"Geolocation" does not need to be capitalized.